stock_picking_transfer_lot_autoassign/wizards/stock_backorder_confirmation.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class StockBackorderConfirmation(models.TransientModel):
+    _inherit = 'stock.backorder.confirmation'
+
+    @api.multi
+    def process_assign_ordered_qty(self):
+        """Autocomplete product qty with producto ordered qty only for
+        products without lots track
+        """
+        self.ensure_one()
+        for pack_operation in self.pick_id.pack_operation_ids:
```

Comment:
`for pack_operation in self.pick_id.pack_operation_ids.filtered(lambda o: not o.pack_lot_ids):`